CondTools/SiPhase2Tracker/BuildFile.xml

```diff
@@ -0,0 +1,10 @@
+<use name="FWCore/Framework"/>
+<use name="FWCore/ParameterSet"/>
+<use name="FWCore/ServiceRegistry"/>
+<use name="CondCore/DBOutputService"/>
+<use name="CondFormats/Common"/>
+<use name="CondFormats/SiPhase2TrackerObjects"/>
+<use name="CondFormats/DataRecord"/>
+<use name="rootrflx"/>
+<!--<use name="boost"/> -->
+<use name="tbb"/>
```

Comment:
why is this needed?

Reply:
@mmusich You mean the BuildFIle.xml or the dependency?

Reply:
just the dependency.